src/Containers/container.jl

```diff
@@ -178,3 +178,8 @@ function container(::Function, ::Any, D::Type)
         "`Containers.container(::Function, indices, ::Type{$(D)})`.",
     )
 end
+
+# Default to dropping names (as before)
+function container(f::Function, names, indices, D)
```

Comment:
This method can probably be moved to line 63, and the docstring updated to reflect the additional argument.